SECURITY-INSIGHTS.yml

```diff
@@ -0,0 +1,38 @@
+header:
+  schema-version: '1.0.0'
+  expiration-date: '2025-01-25T01:00:00.000Z'
+  last-updated: '2024-01-25'
+  last-reviewed: '2024-01-25'
+  project-url: https://github.com/cilium/cilium
```

Comment:
Should this be the github repo or https://cilium.io?

Reply:
From https://github.com/ossf/security-insights-spec/blob/v1.0.0/specification.md#header

>URI for the main project repository that this document refers to.